zanata-model/src/main/java/org/zanata/model/TimeEntityBase.java

```diff
@@ -0,0 +1,41 @@
+package org.zanata.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * @author Alex Eng <a href="aeng@redhat.com">aeng@redhat.com</a>
```

Comment:
This one is definitely going to need a javadoc.  And does the Request's EntityListener belong here?